### PR TITLE
Add default value "targeted" for selinux_policy variable

### PR DESCRIPTION
**Pull request by wrabcak**
Resolves:
https://bugzilla.redhat.com/show_bug.cgi?id=1602897

### CODE REVIEW

**Comment by bachradsusi** [comment]
Could you please rebase your branch to the latest master and repush?

File: selinux-playbook.yml [hunk 1]
```diff
@@ -4,7 +4,6 @@
   become_method: sudo
   become_user: root
   vars:
-    selinux_policy: targeted
```
Comment (bachradsusi) [inline selinux-playbook.yml]: I'd leave in the example playbook as it is so it's clean that you can override it.

Comment (wrabcak) [reply selinux-playbook.yml]: Make sense, will change it.

**Comment by wrabcak** [comment]
We find better solution like this.  We'll take the current value from ansible facts if the variable is not defined.

**Pull request closed without merging**